tools/topology/topology2/production/tplg-targets-ace3.cmake

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Array of "input-file-name;output-file-name;comma separated pre-processor variables"
+list(APPEND TPLGS
+# SDW topology for PTL RVP
+"cavs-sdw\;sof-ptl-rt722\;PLATFORM=ptl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=1,\
```

Comment:
I guess this has dependency on https://github.com/thesofproject/linux/pull/5111 ?

Reply:
yep, it's even described above :-)